app/services/validators/common-validator.js

```diff
@@ -3,6 +3,13 @@
  * three higher level validators: FieldValidator, FieldSetValidator, and UrlPathValidator.
  */
 const validator = require('validator')
+const PRISONER_RELATIONSHIPS = require('../../constants/prisoner-relationships')
```

Comment:
The Java coder in me assumes all caps with underscore means a constant value like a magic string or int , not a object/collection